README.md

```diff
@@ -52,8 +52,8 @@ The libraries need to be added to the `[dependencies]` section of the `Cargo.tom
 
 ```toml
 [dependencies]
-up-rust = { version = "0.6" }
-up-transport-zenoh = { version = "0.7" }
+up-rust = { version = "0.7" }
```

Comment:
Perhaps we can use `cargo add` instead. Then we don't need to update the section every time.

Reply:
You mean replace it with something like
> In order to use the transport, run the following command in your workspace
> ```sh
> cargo add up-rust
> cargo add up-transport-zenoh
> ```

Reply:
Yes